manifests/cf-manifest/operations.d/750-s3-broker.yml

```diff
@@ -0,0 +1,69 @@
+---
+
+- type: replace
+  path: /releases/-
+  value:
+    name: s3-broker
+    version: 0.0.1550145061
+    url: https://s3-eu-west-1.amazonaws.com/gds-paas-build-releases/s3-broker-0.0.1550145061.tgz
+    sha1: d1263c9fc4eaa591aa342aaf0e80a36e14dbad0d
+
+- type: replace
+  path: /addons/name=loggregator_agent/exclude/jobs/-
+  value:
+    name: s3-broker
+    release: s3-broker
+
+- type: replace
+  path: /instance_groups/-
+  value:
+    name: s3_broker
+    azs: [z1, z2]
+    instances: 2
+    vm_type: nano
+    vm_extensions:
+      - s3_broker
+    stemcell: default
+    networks:
+      - name: cf
+    jobs:
+      - name: s3-broker
+        release: s3-broker
+        properties:
+          s3-broker:
+            port: 80
+            basic_auth_username: s3-broker
+            basic_auth_password: ((secrets_s3_broker_admin_password))
+            log_level: INFO
+            aws_region: "((terraform_outputs_region))"
+            resource_prefix: "paas-s3-broker-"
```

Comment:
We should include the `deploy_env` in the prefix. We want to be able to tell which env a bucket belongs to when looking in the AWS console. Otherwise it'll be hard to clean up orphaned buckets from dev environments etc.

Reply:
We add the deploy_env tag to the buckets and the IAM users. Also we don't add the prefix to any other services (RDS, Elasticache), so I'm inclined not to add it here either.

Reply:
We have started adding it to other instances (eg Aiven). We don't add it to Elasticache because we're severely restricted in the length of instance names there (we couldn't even fit a UUID in). I think the RDS broker was one we missed, and it's hard to change now.

Is there a reason not to include it? At the moment it's somewhat tedious to see which environment a bucket belongs to when looking at the list in the S3 console (the tags are many clicks in, and if you're dealing with a long list, that will get old quickly).

Reply:
Nothing against it. We need to to change the IAM policies in account wide terraform to add a "*-" prefix.

Reply:
Or we could use `paas-s3-broker-${env}` as the prefix... which shouldn't require any changes.

Reply:
We usually prefix everything, so I wouldn't put it as a suffix just to avoid an extra small change :)

Reply:
Added the environment - this will apply both to bucket names and IAM usernames. Running down my pipeline to confirm.